src/Spryker/Zed/SynchronizationBehavior/Persistence/Propel/Behavior/SynchronizationBehavior.php

```diff
@@ -184,7 +184,10 @@ public function modifyTable()
             }
         }
 
-        if (isset($parameters['mapping_resource'])) {
+        /**
+         * @deprecated Will be removed without replacement.
```

Comment:
I think we don't need deprecation messages inside the method body. The check method is deprecated already